### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by jrmadsen67**
version is now 0.5.0

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ composer.json:
 
     {
         "require": {
-            "guzzlehttp/oauth-subscriber": "0.4.*"
+            "guzzlehttp/oauth-subscriber": "0.5.*"
```
Comment (GrahamCampbell) [inline README.rst:20]: `^0.5` is probably the syntax we should recommend (instead of `0.5.*`).

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (jrmadsen67) [reply README.rst]: 👍

File: README.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ composer.json:
 
     {
         "require": {
-            "guzzlehttp/oauth-subscriber": "0.5.*"
+            "guzzlehttp/oauth-subscriber": ^0.5"
```
Comment (GrahamCampbell) [inline README.rst:20]: Missing double quote.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (jrmadsen67) [reply README.rst:20]: sorry, I'm actually coding something else while doing this.

**Pull request merged**